api/v1/GameSession.php

```diff
@@ -748,7 +748,7 @@ private function LocalApiRequest(string $apiUrl, int $sessionId, array $postValu
             UrlGeneratorInterface::ABSOLUTE_URL
         );
 
-        $requestHeader = apache_request_headers();
+        $requestHeader = \getallheaders();
```

Comment:
somehow apache_request_headers is not available from frankenphp,  but getallheaders is , and that is just an alias